python/cuml/ensemble/randomforest_common.pyx

```diff
@@ -343,26 +341,20 @@ class BaseRandomForestModel(Base):
                                         predict_proba=predict_proba)
         return preds
 
-    def _get_params(self, deep):
-        params = dict()
-        for key in BaseRandomForestModel.variables:
-            if key in ['handle']:
-                continue
-            var_value = getattr(self, key, None)
-            params[key] = var_value
-        return params
+    def get_param_names(self):
```

Comment:
"_param_names" would seem to be a better name than variables

Reply:
`variables` was already there. I think `_param_names` is much more descriptive but I was trying to change as little as possible.

Reply:
Resolved